### ISSUE TITLE
Package.getActivationCommands is deprecated.

### ISSUE THREAD

**Issue by johnburkhard** [AtomLinter/linter-jshint#86]
Use `activationCommands` instead of `activationEvents` in your package.json
Commands should be grouped by selector as follows:
```json
  "activationCommands": {
    "atom-workspace": ["foo:bar", "foo:baz"],
    "atom-text-editor": ["foo:quux"]
  }
```
```
Package.getActivationCommands (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:810:9)
Package.hasActivationCommands (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:735:20)
<unknown> (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:189:24)
Package.measure (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:167:15)
```

**Comment by johnburkhard** [comment]
Fixed in 0.1.3 release.

**Comment by Arcanemagus** [comment]
Duplicate of #60.
